trillian/ctfe/handlers.go

```diff
@@ -258,7 +262,20 @@ func newLogInfo(
 	}
 
 	once.Do(func() { setupMetrics(instanceOpts.MetricFactory) })
-	knownLogs.Set(1.0, strconv.FormatInt(logID, 10))
+	label := strconv.FormatInt(logID, 10)
+	knownLogs.Set(1.0, label)
+
+	delay := instanceOpts.Config.MaxMergeDelaySec
+	if delay == 0 {
+		delay = 24 * 60 * 60 // 24 hours.
```

Comment:
I am tempted to remove the defaults, and just leave these 0. It will be a responsibility of operators to make sure their configs are in sync with their monitoring alerts.

Reply:
Done.

Reply:
Need to think through the defaults. If we don't have them then all logs inc. existing configs will have to be updated for this to work properly. If we do have them then they may be incorrect for anyone who doesn't have a 24H MMD.